### ISSUE TITLE
Package request: groff

### ISSUE THREAD

**Issue by yegle** [termux/termux-packages#2056]
Just realized that the `cppman` (a tool to check C++ documents from command line) depends on groff and it's not in Termux package list yet.

**Comment by sdrausty** [comment]
Package groff is available at https://sdrausty.github.io/TermuxArch/ in Arch Linux Termux PRoot for the timebeing.   

```
[16:07 home ]$ pacman -S groff
warning: groff-1.22.3-7 is up to date -- reinstalling
resolving dependencies...
looking for conflicting packages...

Packages (1) groff-1.22.3-7

Total Download Size:   1.67 MiB
Total Installed Size:  7.74 MiB
Net Upgrade Size:      0.00 MiB

:: Proceed with installation? [Y/n] 
```

**Comment by mklein994** [comment]
@yegle you can find it at its-pointless' package repo: https://github.com/its-pointless/gcc_termux. Just follow the instructions in the [README][1] to add it, then `apt update && apt install groff`.

The repository hosts quite a few packages that are not currently in the main Termux repository, including groff.

[1]: https://github.com/its-pointless/gcc_termux/blob/73c5df07413e7fc152ad5aa5a03ec97cbe7043a6/README.txt#L2-L12

**Comment by arno01** [comment]
Yeah, it would be good to see that package in the official Termux repo.

``aws cli`` needs the ``groff``.

```
$ aws s3 help

Could not find executable named "groff"
```

**Comment by fancsali** [comment]
Indeed, as @arno01 mentioned, it's a dependency of the very popular `awscli`, and I guess other things too. Not to mention, it is a quite useful tool in its own right.

**Comment by hardikhari96** [comment]
Same here


Could not find executable named "groff"

**Comment by Maxython** [comment]
The `groff` package has been added to termux.  You can install and use safely.
